### PR TITLE
Add argument in UI/pom.xml to allow root

### PR DESCRIPTION
**Pull request by IkaroSales**

### CODE REVIEW

File: UI/pom.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
               <goal>bower</goal>
             </goals>
             <configuration>
-              <arguments>install</arguments>
```
Comment (rvema) [inline UI/pom.xml]: @IkaroSales We have removed bower and PR has been submitted for the same. Hence the below change may not be required

Comment (matthewbeasley) [reply UI/pom.xml]: @IkaroSales @rvema - this is definitely not necessary. Just ensure that your user owns the files in this directory.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Help: rename the adject docker-compose.override.example.yml file to docker-compose.override.yml
 #       then in the override file uncomment the environment tag for the collectors you want to
 #       run. Then add the environment vars you want to use.
-version: '3'
+version: '2'
```
Comment (matthewbeasley) [inline docker-compose.yml]: Why are you downgrading this?

**Comment by matthewbeasley** [comment]
@IkaroSales - please add a description of what all you changed and instructions for how I can test your PR. Also make the requested changes. We are excited about potentially merging this in since kubernetes is on our roadmap.

**Comment by matthewbeasley** [comment]
@IkaroSales please respond to the previous request for information

**Comment by matthewbeasley** [comment]
@IkaroSales - I'm going to close this soon if we don't receive a response from you for the above requests.

**Comment by matthewbeasley** [comment]
Closing this to inactivity

**Pull request closed without merging**